DesktopModules/Vanjaro/Core/Library/App.config

```diff
@@ -1,5 +1,5 @@
 ﻿<configuration>
   <connectionStrings>
-   <add name="SiteSql" connectionString="Data Source=HC22;Initial Catalog=dev950;User ID=sa;Password=20hybridSQL10" providerName="System.Data.SqlClient"/> 
+   <add name="SiteSql" connectionString="Data Source=DESKTOP-9SGJF9E\SA;Initial Catalog=dev_vanjaro_local;User ID=sa;Password=20hybridSQL10" providerName="System.Data.SqlClient"/> 
```

Comment:
This should never be committed. Even though this is local and poses no security risk, we should not commit them with actual values. Let's fix this.